src/Jackett.Common/Definitions/gay-torrentsorg.yml

```diff
@@ -24,6 +24,36 @@ settings:
     type: info
     label: "Search results"
     default: "You can increase the number of search results in your profile.<br />Default is 15."
+  - name: status
+    type: select
+    label: Status
+    default: 0
+    options:
+      0: All
+      1: Active Only
+      2: Dead Only
+      3: Free
+      4: 50%free
+      5: 25%free
+      6: packs
+  - name: sort
+    type: select
+    label: Sort by
+    default: seeds
```

Comment:
default should be data (is that correct, it isn't date?)

Reply:
@LukeSal88  please confirm that `data` is the correct sort value for the date column

Reply:
on the basis that rockbox, nitro, p2pbg, pleasuredome and totallykids also have data as the date column sort parameter,  I'll merge this.